### ISSUE TITLE
interp: unknown GEP

### ISSUE THREAD

**Issue by cornelk** [tinygo-org/tinygo#1110]
here is a stripped down code example of trying to import https://github.com/go-yaml/yaml that fails with error: `<init>: interp: unknown GEP`

```go
package main

import "reflect"

var mapItemType = reflect.TypeOf(MapItem{})

type MapItem struct {
	Key interface{}
}

func main() {}
```

**Comment by aykevl** [comment]
This should be fixed with #1430.

**Comment by deadprogram** [comment]
This has been released with v0.17.0 so now closing. Thanks!
